frontend/src/metabase/public/components/EmbedModal/EmbedModalContent/tests/EmbedModalContent.enterprise.unit.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import { screen, within } from "@testing-library/react";
+
+import { type SetupOpts, setup } from "./setup";
+
+function setupEnterprise(opts?: Partial<SetupOpts>) {
+  setup({
+    ...opts,
+    hasEnterprisePlugins: true,
+  });
+}
+
+describe("EmbedModalContent", () => {
+  describe("Interactive Embedding", () => {
```

Comment:
Test interactive embedding the same way we test it in the `common` test file. This is to ensure the feature isn't leaked to EE build without a valid token (cloud starter)